### ISSUE TITLE
Google Vision API request (Python)?

### ISSUE THREAD

**Issue by koolvic01** [GoogleCloudPlatform/community#212]
I've set up a Google Vision API KEY however while going through the tutorials, i'm confused with how to link an image link to the Vision API request. What do I put for "images:annotate?" Where do I input the image link? How would I incorporate the POST type instead of a get type? 
POST https://vision.googleapis.com/v1/images:annotate?key=YOUR_API_KEY
so far I have
-> baseURL= 'https://vision.googleapis.com/v1/images:annotate?key='+key
-> req = requests.Request(method = 'POST', url =baseURL, params ={'format': 'json'})
-> prepped = req.prepare()
-> fullURL = prepped.url

For example I've got google map api to work with the following: 
		data = requests.get('https://maps.googleapis.com/maps/api/geocode/json?address={}&key=input key here'.format(city), params={'format': 'json'}).json()

If there's a working example python code with Vision API key left out that'd be great. 

I've only imported the following:
import requests
import json
import io

**Comment by koolvic01** [comment]
This site gives good instructions of vision api using client library: https://cloud.google.com/vision/docs/reference/libraries#client-libraries-install-python

**Comment by jmdobry** [comment]
Yes, for Python I recommend using the Google Cloud Client Library for the Cloud Vision API. For example, here's a label detection sample for Python: https://cloud.google.com/vision/docs/detecting-labels#vision-label-detection-python
